src/Avalonia.Controls/TreeView.cs

```diff
@@ -179,6 +179,27 @@ public void ExpandSubTree(TreeViewItem item)
             }
         }
 
+        /// <summary>
+        /// Collapse the specified <see cref="TreeViewItem"/> all descendent
+        /// <see cref="TreeViewItem"/> s.
```

Comment:
nit: space probably should be before the last ` s`

Reply:
yeah sorry, I have a linter that auto formats those sometimes.

Reply:
Addressed.